jigsawpy/__init__.py

```diff
@@ -50,45 +50,42 @@
 ------------------------------------------------------------
  """
 
-from .msh_t import jigsaw_msh_t
-from .jig_t import jigsaw_jig_t
+from jigsawpy.msh_t import jigsaw_msh_t
+from jigsawpy.jig_t import jigsaw_jig_t
 
-from .def_l import *
+from jigsawpy.def_l import *
```

Comment:
In general, I really dislike `import *` but I can see how it's hard to avoid in this case.  My practice would be to do something like:
```py
from jigsawpy import constants
...
myvar[:] = constants.MY_CONST
```
Otherwise, it's very hard to trace where things got imported from.  But you can stick with this syntax if you strongly prefer it.